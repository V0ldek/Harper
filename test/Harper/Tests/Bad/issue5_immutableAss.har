fun n = {
    x = n;
    x += 1;
    return x;
};

main = fun 42;