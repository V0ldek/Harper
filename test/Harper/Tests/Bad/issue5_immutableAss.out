Error: cannot assign to an immutable variable `x`.
During evaluation of:
x := x + 1 ;

Located at line 3 column 5

Execution terminated with an error: runtime error.